parsec/core/invite/organization.py

```diff
@@ -128,20 +128,9 @@ async def failsafe_organization_bootstrap(
             raise
     # If we get an "unknown_command" status, the backend might be too old to know about the "organization_bootstrap" command (API version < 2.7)
     else:
-        if rep["status"] != "unknown_command":
-            return rep
-    # Then we try again with the legacy version
-    if sequester_authority_certificate:
-        raise InviteError("Backend doesn't support sequestered organization")
-    async with apiv1_backend_anonymous_cmds_factory(addr) as anonymous_cmds:
-        assert addr.token is not None, "Token is required to bootstrap an organization"
-
-        return await anonymous_cmds.organization_bootstrap(
-            organization_id=addr.organization_id,
-            bootstrap_token=addr.token,
-            root_verify_key=root_verify_key,
-            user_certificate=user_certificate,
-            device_certificate=device_certificate,
-            redacted_user_certificate=redacted_user_certificate,
-            redacted_device_certificate=redacted_device_certificate,
-        )
+        if (
+            isinstance(rep, OrganizationBootstrapRepUnknownStatus)
+            and rep.status == "unknown_command"
+        ):
+            raise ProtocolError("Unknown command")
```

Comment:
Are we sure about raising this exception ?

What happens in the GUI if we endup here ?

I guess there is a smart move to do here by also removing APIv2 support altogether, this way we would have:
- client supporting APIv3 can do handshake and have the guarantee `organization_bootstrap` is available
- client only suppoting APIv1 will get an error during handshake
- client supporting APIV1 and APIv2 will get an error during APIv2 handshake, then again during APIv1 handshake

In each case, the handshake error is cause by `settle_api_version` raising an `IncompatibleAPIVersionsError` which is handled in a generic way in the GUI (user gets a popup saying his client is incompatible  with the server).

On the other hand if we remove APIv1 but not APIv2, we would still have to handle the case where server provide an APIv2 <2.7 (i.e. without `bootsrap_organization`) and a client having dropped the support for talking to APIv1 (hence the client cannot do an organization bootstrap and we need to provide a special error handling for this weird case...)

what do you think @vxgmichel ?